### PR TITLE
Created fr.json and added French Translation

### PR DESCRIPTION
**Pull request by MeixDev**
Hey ! French Translation of the module by myself (Meï#4242 on Discord if needed).

The only untranslated word is "flags" for the token's flags. As I didn't have a clear translation and this is an option used for interoperability with other modules which may not have been translated, I thought sticking to the english word was the wisest idea.

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "MLT.SettingAutoSyncTargets": "Synchroniser automatiquement les cibles des joueurs",
+  "MLT.SettingAutoSyncTargetsHint": "Si coché, sélectionner ou déselectionner un jeton sélectionnera/déselectionnera aussi ses clones (ou originaux).",
+  "MLT.SettingAutoSyncChatBubbles": "Synchroniser automatiquement les bulles de discussion",
+  "MLT.SettingAutoSyncChatBubblesHint": "Si coché, les bulles de discussion pour un jeton seront aussi visible sur ses clones (ou originaux).",
+  "MLT.SettingCloneModuleFlags": "Les autres modules peuvent changer les flags d'un jeton",
+  "MLT.SettingCloneModuleFlagsHint": "Modules can set custom flags on tokens for their own use. If checked, cloned tokens will inherit such flags from the original. Since the purpose of these flags depends on the module in question, I can't tell you what this option will do, but if cloned tokens are interacting poorly with some other module, you can try changing it.",
```
Comment (grandseiken) [inline lang/fr.json:7]: looks like this got left in by mistake?

**Comment by grandseiken** [comment]
Thanks! Looks good!

**Comment by MeixDev** [comment]
Updated the branch to remove the 2 duplicates. One dumb point for me.

Thanks for your module, it's really useful !

**Comment by grandseiken** [comment]
no problem, thank you for the translation!

**Pull request merged**